src/Homely.AspNetCore.Mvc.Helpers/Controllers/HomeController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Homely.AspNetCore.Mvc.Helpers.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace Homely.AspNetCore.Mvc.Helpers.Controllers
+{
+    [AllowAnonymous]

```

Comment:
should this be removed? 

actually.. is this controller needed, here? shouldn't it be in the test application / template?

Reply:
> should this be removed?
Yep. 🔧 

> actually.. is this controller needed, here?
totally.

```
service.AddMvcCore( .. )
       .AddAHomeController(services, typeof(Startup), "pew pew")...
```